libc-test/build.rs

```diff
@@ -360,6 +361,7 @@ fn test_openbsd(target: &str) {
         "pthread_np.h",
         "sys/syscall.h",
         "sys/shm.h",
+        "iconv.h",
```

Comment:
this break the test on OpenBSD : no `iconv.h` file here

Reply:
I just made #2067 to unbreak OpenBSD

Reply:
Oops, sorry — that's leftovers from the time when the bindings were in the unix mod. The similar problem affects Android, Solari-sh, Redox and a few other targets, I'll submit a PR fixing that.

Reply:
Done: https://github.com/rust-lang/libc/pull/2068